tests/support/commands.ts

```diff
@@ -59,6 +59,31 @@ Cypress.Commands.add('createRuntimeConfig', ({ type } = {}): void => {
   cy.findByRole('button', { name: /save/i }).click();
 });
 
+Cypress.Commands.add('createExampleComponentCatalog', ({ type } = {}): void => {
+  cy.on('fail', e => {
+    console.error(
+      `Example catalog connectors do not appear to be installed.\n${e}`
+    );
+    throw new Error(
+      `Example catalog connectors do not appear to be installed.\n${e}`
+    );
+  });
+
+  cy.findByRole('tab', { name: /component catalogs/i }).click();
+  cy.findByRole('button', { name: /create new component catalog/i }).click();
+
+  if (type === 'kfp') {
+    cy.findByRole('menuitem', { name: /kubeflow pipelines/i }).click();
+  } else {
+    cy.findByRole('menuitem', { name: /apache airflow/i }).click();
+  }
+
+  cy.findByLabelText(/^name/i).type('Example Components');
```

Comment:
we should probably make this step check if one already exists first in case we end up using this in multiple tests in the future

Reply:
I'm currently removing the example catalog at the very end of the `afterEach` block for the pipeline tests (similar to how test runtime configs are handled). I see what you're saying, but I wonder if we should just leave this to be dealt with when the time comes? Idk, I could go either way

Regardless, I am going to change the name of the catalog to be something like "Test Example Components" to reduce the likelihood of collisions with existing catalogs when running locally.

Reply:
ah I see, I though that code was run once at the end, not per test. In that case ignore my comment, this way is good